src/Microsoft.DotNet.Build.Tasks/PackageRestoreLogger.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class PackageRestoreLogger : ILogger
+    {
```

Comment:
How would this be consumed?

Reply:
This is being consumed by any restore that happens via packageresolve.targets

Reply:
In the context of official builds, this becomes less clear /cc @chcosta

Reply:
This will require a buildtools update in all repos.

Reply:
i'd imagine that any repo which wants to validate it's successfully accomplished phase2, would update buildtools, and either run an official build with the "enable PackageRestoreLogger" property set, or run a local build with the same property set.

Reply:
yes currently the property is DotNetRestoreLog

Reply:
changed it to EnablePackageRestoreLogger.